fv3_cap.F90

```diff
@@ -406,7 +408,23 @@ subroutine InitializeAdvertise(gcomp, rc)
       line=__LINE__, file=__FILE__)) call ESMF_Finalize(endflag=ESMF_END_ABORT)
 
     ! Read in the FV3 coupling interval
-    call clock_cplIntval(gcomp, CF)
+    call ESMF_ConfigGetAttribute(config=CF, value=medAtmCouplingIntervalSec, &
```

Comment:
I thought we don't need the code related to "atm_coupling_interval_sec" any more. Can you remove this section, my understanding is the code should still work.

Reply:
We still do need this option. I tried to remove that entire section (former clock_cplIntval subroutine) but coupled tests fail. Removing this option will require some changes in how we handle clocks in the cap.

Reply:
I see. So this is where the timestep is not available as Rocky mentioned. But the internal timestep is available at ModelAdvance, I am thinking of removing the atm_coupling_interval_sec and use the timestep at ModelAdvance to reset the clock, what do you think? Currently the atm_coupling_interval_sec has to be consistent with the inner time interval in the run sequence.

Reply:
I agree that we should take a closer look at how clocks are defined and/or modified in the cap, and maybe see if we can eliminate the need for the cap to keep it's own local clock (`clock_fv3`). But I would like to keep this PR as it is and try to  address this issue in one of the future PRs. The current logic in the code related to clocks is not trivial, and it's not easy follow what's really going on.